### DISCUSSION TITLE
Would like to hire someone to create a basic daycare management app

### DISCUSSION THREAD

**Discussion by SMTJ305** [Budibase/budibase#11744] [General]
Hi, as mentioned, I would like to hire someone who could create a fairly basic daycare management app asap on budibase. i have created the base but since im not aware of how to include many of the functions needed for it, i havent been able to go forward w the developing. if anyone is interested, please let me know

**Comment by zedascouves** [comment]
Hi, I may be able to help. Can you provide any more detail or send an email to `me@duarte.vg?`

**Comment by JayP718** [comment]
Hi,

Feel free to reach out. I've built multiple plugins, won the Budibase hackathon and built multiple Budibase applications.

Jay@a3datasolutions.com
